isis/src/base/objs/FileName/FileName.cpp

```diff
@@ -622,8 +623,7 @@ namespace Isis {
                     foundFile.mid(truncateRange.second + 1);
 
       }
-
-      QDate fileDate = QDate::fromString(foundFile, fileQDatePattern);
+      QDate fileDate = QLocale().toDate(foundFile, fileQDatePattern);
```

Comment:
This member was designed to work with US English locale ONLY. Would it be better to force this call to use US English, and not rely on the Application to set it correctly?